apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -1,20 +1,295 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { ErrorBoundary } from '../../components/error-boundary/ErrorBoundary';
-import { TopValidatorsCard } from '../../components/top-validators-card/TopValidatorsCard';
+import { is, SuiObject } from '@mysten/sui.js';
+import { lazy, Suspense, useState, useMemo } from 'react';
 
+import Pagination from '../../components/pagination/Pagination';
+
+import { ErrorBoundary } from '~/components/error-boundary/ErrorBoundary';
+import { StakeColumn } from '~/components/top-validators-card/StakeColumn';
+import { useGetObject } from '~/hooks/useGetObject';
+import { DelegationAmount } from '~/pages/validator/DelegationAmount';
+import {
+    VALIDATORS_OBJECT_ID,
+    type ValidatorState,
+} from '~/pages/validator/ValidatorDataTypes';
+import { Banner } from '~/ui/Banner';
+import { Card } from '~/ui/Card';
 import { Heading } from '~/ui/Heading';
+import { ImageIcon } from '~/ui/ImageIcon';
+import { ValidatorLink } from '~/ui/InternalLink';
+import { PlaceholderTable } from '~/ui/PlaceholderTable';
+import { Stats } from '~/ui/Stats';
+import { TableCard } from '~/ui/TableCard';
+import { TableHeader } from '~/ui/TableHeader';
+import { Text } from '~/ui/Text';
+import { getName } from '~/utils/getName';
+import { getStakedPercent } from '~/utils/getStakedPercent';
+
+const NUMBER_OF_VALIDATORS = 20;
+
+const ValidatorMap = lazy(
+    () => import('../../components/validator-map/ValidatorMap')
+);
 
 function ValidatorPageResult() {
+    const { data, isLoading, isSuccess, isError } =
+        useGetObject(VALIDATORS_OBJECT_ID);
+
+    const [validatorsPageNumber, setValidatorsPageNumber] = useState(1);
+
+    const validatorsData =
+        data &&
+        is(data.details, SuiObject) &&
+        data.details.data.dataType === 'moveObject'
+            ? (data.details.data.fields as ValidatorState)
+            : null;
+
+    const validatorsStats = useMemo(() => {
```

Comment:
Same here, would not create this view model.

Reply:
done